lib/core/templates.html

```diff
@@ -464,6 +464,19 @@ <h3 class="categoryButton" data-category="{{name}}">{{name}}</h3>
 			</div>
 		</script>
 		<!--/imgurgifvUI-->
+		<!--pornbotUI-->
+		<script id="imgurgifvUI" type="application/x-template">
```

Comment:
we should maybe have a general-use video player instead of copying these players over and over.